problem-spring-common/src/test/java/org/zalando/problem/spring/common/AdviceTraitLoggingTest.java

```diff
@@ -39,40 +30,35 @@ void reset() {
         TestLoggerFactory.clear();
     }
 
-    static Status[] data() {
-        return Status.values();
+    static HttpStatus[] data() {
+        return HttpStatus.values();
     }
 
     @ParameterizedTest
     @MethodSource("data")
-    void shouldLog4xxAsWarn(final Status status) {
-        assumeTrue(status.getStatusCode() / 100 == 4);
-        unit.create(status, new NoHandlerFoundException("GET", "/", new HttpHeaders()), mock(NativeWebRequest.class));
+    void shouldLog4xxAsWarn(final HttpStatus status) {
+        assumeTrue(status.is4xxClientError());
+        unit.log(new RuntimeException("Test message"), null, status);
```

Comment:
There idea here was that the test verifies that log is indeed called as part of the normal `create` method.